content/en/docs/concepts/storage/volumes.md

```diff
@@ -11,221 +11,196 @@ weight: 10
 
 <!-- overview -->
 
-On-disk files in a container are ephemeral, which presents some problems for
-non-trivial applications when running in containers. One problem
-is the loss of files when a container crashes. The kubelet restarts the container
-but with a clean state. A second problem occurs when sharing files
-between containers running together in a `Pod`.
-The Kubernetes {{< glossary_tooltip text="volume" term_id="volume" >}} abstraction
-solves both of these problems.
-Familiarity with [Pods](/docs/concepts/workloads/pods/) is suggested.
+File storage is useful when building stateful applications. Use _volumes_ to declaratively manage storage for containers. The Kubernetes {{< glossary_tooltip text="volume" term_id="volume" >}} abstraction allows files to persist even beyond the lifetime of a pod. Volumes also allow different containers within a pod to share storage and to access configuration.
+
+Familiarity with [Pods](/docs/concepts/workloads/pods/pod/) is suggested.
 
 <!-- body -->
 
-## Background
+## Introduction
 
-Docker has a concept of
-[volumes](https://docs.docker.com/storage/), though it is
-somewhat looser and less managed. A Docker volume is a directory on
-disk or in another container. Docker provides volume
-drivers, but the functionality is somewhat limited.
+By default, everything with the filesystems of a container is ephemeral. During startup, each pod in a container gets a filesystem created based on the layers in its {{< glossary_tooltip text="container image" term_id="image" >}}. When a container is restarted or destroyed, that root filesystem, including any modifications to it, is lost.
 
-Kubernetes supports many types of volumes. A {{< glossary_tooltip term_id="pod" text="Pod" >}}
-can use any number of volume types simultaneously.
-Ephemeral volume types have a lifetime of a pod, but persistent volumes exist beyond
-the lifetime of a pod. Consequently, a volume outlives any containers
-that run within the pod, and data is preserved across container restarts. When a
-pod ceases to exist, the volume is destroyed.
+A Kubernetes volume's lifecycle is tied to the *pod* that encloses it. The pod maintains the volume as different containers are started or stopped. Multiple volumes of any type may be connected to a pod. The [PersistentVolume](/docs/concepts/storage/persistent-volumes/) abstraction builds on the volume concept to provide storage that outlasts the lifetime of a pod.
```

Comment:
How about:
```suggestion
A Kubernetes volume's lifecycle is tied to the *pod* that encloses it. The pod maintains the volume as different containers are started or stopped. Multiple volumes of any type may be connected to a pod. The [PersistentVolume](/docs/concepts/storage/persistent-volumes/) abstraction builds on the volume concept to let pods access storage that outlasts the lifetime of a pod.
```

Reply:
I was attempting to avoid passive voice, but in this case it's the best option.